apps/transport/lib/opendatasoft/url_extractor.ex

```diff
@@ -0,0 +1,185 @@
+defmodule Opendatasoft.UrlExtractor do
```

Comment:
To make this file accessible you need to create a `apps/transport/lib/opendatasoft.ex` like `apps/transport/lib/mailjet.ex`